pkg/api/login.go

```diff
@@ -340,8 +353,13 @@ func (hs *HTTPServer) Logout(c *contextmodel.ReqContext) {
 
 	authn.DeleteSessionCookie(c.Resp, hs.Cfg)
 
-	if setting.SignoutRedirectUrl != "" {
-		c.Redirect(setting.SignoutRedirectUrl)
+	rdUrl := setting.SignoutRedirectUrl
```

Comment:
the variable name can be improved.

Reply:
rdUrl --> i meant redirectUrl. Shall i explicitly put redirectUrl?

Reply:
I think redirectUrl is quite common, but for example the oauth2-proxy just calls it "rd"
https://oauth2-proxy.github.io/oauth2-proxy/docs/features/endpoints/

Reply:
@DanielHons This is variable Name only, It won't be passed to any query parameter. I was just asking if we can name it better for better readability.